### PR TITLE
[GHA][Model Hub Tests] Use openvino-tokenizers in model hub tests

### PR DESCRIPTION
**Pull request by rkazants**
**Details:** Use openvino-tokenizers in model hub tests.

**Ticket:** TBD

### CODE REVIEW

File: .github/workflows/job_pytorch_models_tests.yml [hunk 1]
```diff
@@ -98,7 +104,9 @@ jobs:
           self-hosted-runner: ${{ contains(inputs.runner, 'aks') }}
 
       - name: Install OpenVINO Python wheels
-        run: python3 -m pip install ${INSTALL_DIR}/tools/openvino-*
+        run: |
+          python3 -m pip install ${INSTALL_DIR}/tools/openvino-*
+          python3 -m pip install ${INSTALL_DIR}/openvino-tokenizers.whl
```
Comment (ilya-lavrenov) [inline .github/workflows/job_pytorch_models_tests.yml:109]: I suppose wheel has different name:
image

Comment (rkazants) [reply .github/workflows/job_pytorch_models_tests.yml]: Fixed. Let us see directory structure for OV Tokenizers artifacts also.

File: .github/workflows/job_pytorch_models_tests.yml [hunk 2]
```diff
@@ -98,7 +104,9 @@ jobs:
           self-hosted-runner: ${{ contains(inputs.runner, 'aks') }}
 
       - name: Install OpenVINO Python wheels
-        run: python3 -m pip install ${INSTALL_DIR}/tools/openvino-*
+        run: |
+          python3 -m pip install ${INSTALL_DIR}/tools/openvino-*
+          python3 -m pip install ${INSTALL_DIR}/openvino-tokenizers-*
```
Comment (rkazants) [inline .github/workflows/job_pytorch_models_tests.yml:109]:
```suggestion
          python3 -m pip install ${INSTALL_DIR}/openvino_tokenizers-*
```

File: .github/workflows/job_tensorflow_hub_models_tests.yml [hunk 3]
```diff
@@ -97,7 +103,9 @@ jobs:
           self-hosted-runner: ${{ contains(inputs.runner, 'aks') }}
 
       - name: Install OpenVINO Python wheels
-        run: python3 -m pip install ${INSTALL_DIR}/tools/openvino-*
+        run: |
+          python3 -m pip install ${INSTALL_DIR}/tools/openvino-*
+          python3 -m pip install ${INSTALL_DIR}/openvino-tokenizers-*
```
Comment (rkazants) [inline .github/workflows/job_tensorflow_hub_models_tests.yml:108]:
```suggestion
          python3 -m pip install ${INSTALL_DIR}/openvino_tokenizers*
```

File: .github/workflows/job_pytorch_models_tests.yml (hunk 1 above)
Comment (rkazants) [reply .github/workflows/job_pytorch_models_tests.yml]: Located after unzip in `/install/openvino_tokenizers-2024.0.0.0-py3-none-manylinux_2_31_x86_64.whl`

File: .github/workflows/job_pytorch_models_tests.yml [hunk 4]
```diff
@@ -98,7 +104,9 @@ jobs:
           self-hosted-runner: ${{ contains(inputs.runner, 'aks') }}
 
       - name: Install OpenVINO Python wheels
-        run: python3 -m pip install ${INSTALL_DIR}/tools/openvino-*
+        run: |
+          python3 -m pip install ${INSTALL_DIR}/tools/openvino-*
+          python3 -m pip install ${INSTALL_DIR}/openvino_tokenizers*
```
Comment (rkazants) [inline .github/workflows/job_pytorch_models_tests.yml:109]:
```suggestion
          python3 -m pip install ${INSTALL_DIR}/openvino_tokenizers-*
```

File: .github/workflows/job_tensorflow_hub_models_tests.yml [hunk 5]
```diff
@@ -97,7 +103,9 @@ jobs:
           self-hosted-runner: ${{ contains(inputs.runner, 'aks') }}
 
       - name: Install OpenVINO Python wheels
-        run: python3 -m pip install ${INSTALL_DIR}/tools/openvino-*
+        run: |
+          python3 -m pip install ${INSTALL_DIR}/tools/openvino-*
+          python3 -m pip install ${INSTALL_DIR}/openvino_tokenizers*
```
Comment (rkazants) [inline .github/workflows/job_tensorflow_hub_models_tests.yml:108]:
```suggestion
          python3 -m pip install ${INSTALL_DIR}/openvino_tokenizers-*
```

**Comment by ilya-lavrenov** [comment]
Let's also update https://github.com/openvinotoolkit/openvino/blob/master/.github/components.yml:
- Remove `TEMPLATE` from `TEMPLATE` and `IE_TESTS`
- Add to `build` sections for `PyTorch_FE` and `TF_FE`

**Comment by rkazants** [comment]
> Let's also update https://github.com/openvinotoolkit/openvino/blob/master/.github/components.yml:
> 
> * Remove `TOKENIZERS` from `TEMPLATE` and `IE_TESTS`
> * Add to `build` sections for `PyTorch_FE` and `TF_FE`

Done

**Review by ilya-lavrenov — APPROVED** [review]

**Pull request merged**